<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/overlay/overlay.png</normaloff>../res/overlay/overlay.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: grey;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="_graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/overlay.png);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/buttons/startS_yellow.png);
}

QPushButton:pressed {
	border-image: url(:/buttons/startS_green.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_CESout">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/CES_output_yellow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_audio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/audio_input_yellow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_logo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_corp">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/mindalive.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_country">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/ca.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_up">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/buttons/upButton_yellow.png);
}

QPushButton:pressed {
	border-image: url(:/buttons/upButton_green.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_down">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/buttons/downButton_yellow.png);
}

QPushButton:pressed {
	border-image: url(:/buttons/downButton_green.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_MET">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/MET_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_20min">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session group/20m_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_45min">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session group/45m_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_3hour">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session group/3h_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_userDef">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session group/userDefine_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/buttons/powerButton_yellow.png);
}

QPushButton:pressed {
	border-image: url(:/buttons/powerButton_green.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_intensity">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/int_tag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="misc_powersig">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/misc/powerSig_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_sDelta">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/sDel_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_delta">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/del_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_theta">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/theta_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_alpha">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/alpha_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_beta1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/beta1_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_beta2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/beta2_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="freq_100hz">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/session freq/100hz_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ces_spulse">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/CES mode/spulse_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ces_dutycycle">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/CES mode/dutycycle_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ces_left">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/CES mode/leftCES_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ces_right">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/CES mode/rightCES_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tdcs_l1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level1_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level2_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level3_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level4_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level5_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level6_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level7_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/level8_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/tDCS/tDCS_default.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
